style.css

```diff
@@ -0,0 +1,854 @@
+  opacity: 0;
+  transition: opacity 9s ease-in-out;
+  border-radius: 10px;
+  background: linear-gradient(45deg, #f00, #ff7300, #fffb00, #48ff00, #00ffd5, #002bff, #7a00ff, #ff00c8, #f00);
+}
+
+button:hover {
+  background: linear-gradient(45deg, #f00, #ff7300, #fffb00, #48ff00, #00ffd5, #002bff, #7a00ff, #ff00c8, #f00);
+  opacity: 1;
+  outline: 0;
+  color: var(--black-color);
+  line-height: 20px;
+  position: relative;
+  text-align: center;
+  letter-spacing: 1px;
+  text-decoration: none;
+  font-weight: 400px;
+  font-family: sans-serif;
+  text-transform: uppercase;
+}
+
+button:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.social-link:hover {
+  color: #5e6c84;
+}
+
+.navbar-links > li > a:hover {
+  color: var(--black-color);
+  animation: glowing 20s linear infinite;
+  opacity: 0;
+  transition: opacity 9s ease-in-out;
+  border-radius: 10px;
+  background: linear-gradient(45deg, #f00, #ff7300, #fffb00, #48ff00, #00ffd5, #002bff, #7a00ff, #ff00c8, #f00);
+}
+
+.social-icon > a > img:hover {
+  box-shadow: 1px 2px 4px 1px var(--button-disabled-text-color);
+  transform: scale(1);
+  background-color: var(--gray--light-color);
+  transition: transform 3s;
+}
+
+/* end */
+
+@media screen and (min-width: 768px) {
+  /* images */
+  .img1 {
+    content: url("./images-portfolio-desktop/Snapshoot\ Portfolio.png");
+  }
+
+  .img2 {
+    content: url("./images-portfolio-desktop/Snapshoot\ Portfolio-1.png");
+  }
+
+  .img3 {
+    content: url("./images-portfolio-desktop/Snapshoot\ Portfolio-2.png");
+  }
+
+  .img4 {
+    content: url("./images-portfolio-desktop/Snapshoot\ Portfolio-3.png");
+  }
+
+  .container {
+    border: none;
+    width: 100%;
+    background-color: #f4f5f7;
+  }
+
+  header {
+    width: 100%;
+    position: fixed;
+    top: -3vh;
+    background: white;
+    z-index: 1;
+  }
+
+  .navbar {
+    margin-top: -2%;
+    animation: shownav 5s linear;
+  }
+
+  .logo {
+    animation-play-state: running;
+    align-self: center;
+    animation-duration: 0.7s;
+  }
+
+  .bar-toggle {
+    display: none;
+  }
+
+  .navbar-links {
+    display: flex;
+    min-width: 35%;
+    justify-content: space-between;
+  }
+
+  .navbar-links > li > a {
+    text-decoration: none;
+  }
+
+  .small-container {
+    margin-top: 5.5%;
+    background: url(./images-portfolio-desktop/Header\ bg.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-color: white;
+    height: 100vh;
+    padding: 100px 80px;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin-bottom: 2px;
+  }
+
+  .skills > span > img {
+    position: relative;
+    top: 10px;
+    left: -80px;
+  }
+
+  .card-info {
+    padding: 0 1%;
+    grid-template: repeat(4, auto) / minmax(0, 1fr);
+  }
+
+  .card-item {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    padding: 16px;
+    padding-bottom: 22px;
+  }
```

Comment:
- [ ] :thinking: In my opinion in order to stick to the Figma, we should have enough margin to the sides of the card as shown below:
- [ ]  Your design:
![Screenshot from 2023-01-18 18-32-54](https://user-images.githubusercontent.com/97834160/213248759-bfa882b3-e3a7-4212-8652-118d0816cf72.png)



- Figma:

![Screenshot from 2023-01-18 18-54-31](https://user-images.githubusercontent.com/97834160/213249170-02462c70-4f30-45ad-ba3e-aced535a2eae.png)